### PR TITLE
docs: Compliance Working Group details addition to security tag landing page

### PR DESCRIPTION
**Pull request by AleJo2995**

### CODE REVIEW

**Comment by PushkarJ** [comment]
Please hold as a few discussions are ongoing in preparation for merging this

File: compliance/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+authoring and assessment.
+
+The key areas of the Compliance WG include:
+
+- Building a knowledge base (in github and possibly other tools) and case studies on the How, What, Why and When of operating a cloud native environment within the requirements of legal and regulatory 
+entities that govern clouds, specific industries, and more generally data and public/consumer usage. These requirements are often NOT JUST  technical security concern - compliance activities and 
+requirements span human activities and performance, system availability and reliability, the combined human and technical aspects of continuity of operations, defining and monitoring data location as 
+well as sovereignty and provenance of the regulated environment components and data
+
+- Generating specific examples of compliance as code, normalized templates, and tools for automating these both technical and non-technical requirements, control assessment, data analyses, audit and
```
Comment (PushkarJ) [inline compliance/README.md:24]:
```suggestion
- Generating specific examples of compliance as code, normalized templates, and tools for automating these both technical and non-technical requirements, control assessment, data analysis, audit and 
```

File: compliance/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Generating specific examples of compliance as code, normalized templates, and tools for automating these both technical and non-technical requirements, control assessment, data analyses, audit and 
+compliance remediation workflows that specifically benefit CNCF projects and their community of users.
+
+- Reviewing industry and governmental standards - eg NIST, PCI, HIPAA, etc - from a cloud native perspective and serving as Subject Matter Experts in the CNCF community for how projects should implement 
+and support these compliance-specific requirements as first class citizen to enable broad adoption of the best practices by commercial, non-profit, governmental, and humanitarian organizations.
+
+## Responsibilities
+
+- Users/personas/needs/customer demands for industry and regulatory compliance (both human and technical)
+- Identifications of areas of focus eg human workflows, automated workflows, analytical tools, audit and assessment tools, technical security controls that cut across components and systems and clouds, etc
```
Comment (PushkarJ) [inline compliance/README.md:33]:
```suggestion
- Identifications of areas of focus e.g. human workflows, automated workflows, analytical tools, audit and assessment tools, technical security controls that cut across components and systems and clouds, etc
```

File: compliance/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- Framework for evaluation, audit and reporting - how do products and tools demonstrate compliance?
+- Training and automation - what is missing, what is difficult to understand, what knowledge gaps are there?
+- Work on integrating common tooling across different projects, particularly where that tooling is a CNCF project (but the targets may not be)
+- Cross project focus on the projects and efforts the CNCF is funding, helping projects identify needs and providing subject matters to assist
```
Comment (PushkarJ) [inline compliance/README.md:37]:
```suggestion
- Cross project focus on the projects and efforts the CNCF is funding, helping projects identify needs and providing subject matter expertise to assist
```

File: compliance/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- Recommendations of integrating security tooling with compliance tooling and processes - making both the synergies and unique separations of concern explicit and achieving community consensus.
+- Growing CNCF external relationships with interested parties, eg NIST and other compliance standards bodies such as FINOS, OSCAL, OpenSSF
```
Comment (PushkarJ) [inline compliance/README.md:39]:
```suggestion
- Growing CNCF external relationships with interested parties, e.g. NIST and other compliance standards bodies such as FINOS, OSCAL, OpenSSF
```

File: compliance/README.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Lou DeGenaro
+
+Manjiree Gadgil
+
+Vikas Agarwal
+
+Yuji Watanabe
+
+Takumi Yanagawa
```
Comment (PushkarJ) [inline compliance/README.md:64]: Can we add everyone's GitHub handles in front of the names?

**Review by PushkarJ — CHANGES_REQUESTED** [review]
Just added some minor language changes that make the charter clearer to understand :) happy to chat more if there is anything that needs discussion.

File: README.md [hunk 6]
```diff
@@ -149,6 +149,18 @@ seen [here](governance/related-groups/)
 
 ### On-going projects
 
+#### Compliance Working Group
+
+
+[Compliance Working Group](./compliance) is cross-disciplinary and focused on bridging purely technical issues to broader legal and regulatory workflows: not only the prevention of system breaches, but also considering supply chain, operators, data and AI failures while simultaneously considering auditability, non-repudiation, legally required forensic evidence, etc. - across all the various activities required of cloud native operators of all sizes. The Compliance WG plans to curate tools for evidence collection, chain-of-custody in audits, as well as automated workflows for continuous compliance authoring and assessment.
+
+Weekly meetings at 10:00 AM ET (60 min)
```
[the remark this answers is not in the document]
Comment (AleJo2995) [reply README.md:157]: How cool 👍

**Comment by PushkarJ** [comment]
Thanks @AleJo2995 can you please fix the linting errors?

**Comment by PushkarJ** [comment]
I committed a couple of residual suggestions too.

**Comment by AleJo2995** [comment]
Also, if you see Spelling check is complaining about names and last names? How should I fix this?

image

**Comment by PushkarJ** [comment]
I ran the checks for you now. 

Try cspell enable and disable for ignoring names https://cspell.org/configuration/document-settings/

**Comment by AleJo2995** [comment]
@PushkarJ I have made the changes for the first to checks to pass, but for the one with the links I am not being able to solve it. What I am curious about is why it is failing if I haven't changed anything there 😅

**Comment by PushkarJ** [comment]
You can ignore the links errors. Only three linting errors remaining :) after that we are good to go

**Comment by AleJo2995** [comment]
@PushkarJ I have done the changes for the trailing spaces that were problematic. I think this is now good to go. Thanks 👍

**Review by PushkarJ — APPROVED** [review]

**Review by mnm678 — APPROVED** [review]

**Pull request merged**